akka-contrib/src/main/scala/akka/contrib/pattern/DistributedPubSubMediator.scala

```diff
@@ -108,6 +110,26 @@ object DistributedPubSubMediator {
     def this(path: String, msg: Any) = this(path, msg, allButSelf = false)
   }
 
+  sealed abstract class GetTopics
+  @SerialVersionUID(1L)
+  case object GetTopics extends GetTopics
+
+  /**
+   * Java API
+   */
+  def getTopicsInstance: GetTopics = GetTopics
+
+  @SerialVersionUID(1L)
+  final case class CurrentTopics(topics: Map[String, ActorRef]) {
```

Comment:
why should we expose the ActorRef in the reply?
shouldn't it be a `Set[String]`?